docs/bokeh/source/docs/first_steps.rst

```diff
@@ -56,7 +56,7 @@ First steps guides
 Follow these guides to quickly learn about the most important features and
 capabilities of Bokeh.
 
-The first steps guides are for anybody who is new to Bokeh. The only
```

Comment:
@DimitarSivrev this is intentional and correct. This is a collection of guides, relating to first steps learning Bokeh.  "First steps guides" please revert this one change.

cc @tcmetzger should we start hyphenating "first-steps" when it is used before guides?

Reply:
@bryevdv That is a good idea, that'll make it easier to read!